<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Seleccione una clase y haga click en el plano para ingresar un patrón de entrenamiento</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="plot_area"/>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Datos de entrenamiento</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="11" column="1" colspan="2">
           <widget class="QPushButton" name="clear_button">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Limpiar</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Clase seleccionada</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1" colspan="2">
           <widget class="QPushButton" name="begin_button">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Iniciar</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QTableWidget" name="result_table">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>75</number>
            </attribute>
            <row>
             <property name="text">
              <string>N1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>N2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>N3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>N4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>N5</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>MSE</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>w1</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>w2</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>b</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="1" column="3">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="2">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="3">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="2">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="3">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="4" column="1">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="4" column="2">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="4" column="3">
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="target_error">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0.08</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Iteraciones máximas</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="max_iterations">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>200</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Razón de aprendizaje</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="class1">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 0, 255);
color: white;
</string>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="class2">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 187, 255);
color: white;
</string>
              </property>
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="class3">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 190, 32);
color: white;</string>
              </property>
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="class4">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(227, 178, 0);
color: white;
</string>
              </property>
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="class5">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(220, 0, 0);
color: white;
</string>
              </property>
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="learning_rate">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0.02</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="iteration_label">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Iteración</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="current_class">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font-style: italic;</string>
            </property>
            <property name="text">
             <string>Seleccione una clase</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>M-MSE</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Error objetivo</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Modo</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="mmse_label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="selector_mode">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Aproximador</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Una clase</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
